src/service/core/config/objects.py

```
@@ -70,6 +70,30 @@
+                        'medium': 'Memory',
+                        'sizeLimit': '64Gi'
+                    }
+                }
+            ],
+            'containers': [
+                {
+                    'name': '{{USER_CONTAINER_NAME}}',
+                    'volumeMounts': [
+                        {
+                            'name': 'shm',
+                            'mountPath': '/dev/shm'
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/dev/shm"

(S108)